pkg/acsengine/k8s_versions.go

```diff
@@ -258,6 +283,31 @@ var KubeConfigs = map[string]map[string]string{
 		"gchighthreshold": strconv.Itoa(DefaultKubernetesGCHighThreshold),
 		"gclowthreshold":  strconv.Itoa(DefaultKubernetesGCLowThreshold),
 	},
+	common.KubernetesVersion1Dot6Dot12: {
+		"hyperkube":       "hyperkube-amd64:v1.6.12",
+		"dashboard":       "kubernetes-dashboard-amd64:v1.6.3",
+		"exechealthz":     "exechealthz-amd64:1.2",
+		"addonresizer":    "addon-resizer:1.7",
+		"heapster":        "heapster-amd64:v1.3.0",
+		"dns":             "k8s-dns-kube-dns-amd64:1.14.5",
+		"addonmanager":    "kube-addon-manager-amd64:v6.4-beta.2",
+		"dnsmasq":         "k8s-dns-dnsmasq-nanny-amd64:1.14.5",
+		"pause":           "pause-amd64:3.0",
+		"tiller":          DefaultTillerImage,
+		"windowszip":      "v1.6.11-1intwinnat.zip", // TODO: cleanup
```

Comment:
@CecileRobertMichon you can delete the `"windowszip"` property entirely here because we're no longer supporting Windows k8s < 1.7 (RS3 doesn't work with 1.6)